msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/results/SignInResult.kt

```diff
@@ -101,3 +117,24 @@ interface SignInResendCodeResult : Result {
         val channel: String,
     ) : SignInResendCodeResult, Result.SuccessResult(nextState = nextState)
 }
+
+// TODO should we call this SignInMFA or just MFA? MFA can only happen during sign in, so it's redundant?
```

Comment:
Let's discuss this.

Reply:
I don't have a strong opinion on this but since all the interfaces start with SignIn maybe its better to follow this convention for consistency